kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/GeneralErrorCorrectionAction.java

```diff
@@ -352,7 +361,6 @@ private void dissociateEntryByLine(GeneralErrorCorrectionDocument doc, GECSource
         // Effect the changes
         if (relsToDelete.size() > 0) {
             doc.getGecEntryRelationships().removeAll(relsToDelete);
-            getBusinessObjectService().delete(relsToDelete);

```

Comment:
From previous commit: the collection will manage deleted objects automatically no reason to force it through BOService.delete().